src/sentry/models/apiapplication.py

```diff
@@ -32,7 +32,7 @@ class ApiApplication(Model):
     owner = FlexibleForeignKey('sentry.User')
     name = models.CharField(
         max_length=64, blank=True,
-        default=lambda: petname.Generate(2, ' ').title())
+        default=lambda: petname.Generate(2, ' ', letters=1).title())
```

Comment:
Damn, I actually misinterpreted this. Going to fix on master. This is "max length".